toolbox/load_agol_layer/src/load_agol_layer.js

```diff
@@ -0,0 +1,70 @@
+const fs = require("fs");
+const { program, Option } = require("commander");
+const { arcgisToGeoJSON } = require("@terraformer/arcgis");
+const {
+  getEsriToken,
+  getEsriLayerUrl,
+  getEsriJson,
+  handleFields,
+  makeUniformMultiPoly,
+  makeHasuraRequest,
+  reduceGeomPrecision,
+} = require("./utils");
+const { LAYERS } = require("./settings");
+
+/**
+ * CLI
+ */
+program.addOption(
+  new Option("-l, --layer <name>", "layer name")
+    .choices(Object.keys(LAYERS))
+    .makeOptionMandatory()
+);
+program.parse();
+const args = program.opts();
+
+/**
+ * Main function to upsert an AGOL layer into the DB
+ */
+const main = async ({ layer: layerName }) => {
+  console.log(`Processing ${layerName}`);
+  const layerConfig = LAYERS[layerName];
+  const { token } = await getEsriToken();
+  layerConfig.query_params.token = token;
+  const layerUrl = getEsriLayerUrl(layerConfig);
+  const esriJson = await getEsriJson(layerUrl);
+
+  /**
+   * Although the ArcGIS REST API can return geojson directly, the resulting geometries
+   * are malformed. For this reason, we use terraformer package, which is an Esri
+   * product.
+   * 
+   * The issue is discussed in the Esri community, here:
+   * https://community.esri.com/t5/arcgis-online-questions/agol-export-to-geojson-holes-not-represented-as/td-p/1008140
+   */
+  let geojson = arcgisToGeoJSON(esriJson);
+
+  if (esriJson.geometryType.toLowerCase().includes("polygon")) {
```

Comment:
the function already does a check to see if the type is polygon, so at first glance this seems a little repetitive?

Reply:
oh i think i get it, bc for other valid [geometryTypes](https://developers.arcgis.com/arcgis-rest-js/api-reference/arcgis-rest-request/GeometryType/) we might have in our db like maybe points or lines we wouldnt need to run the makeUniformMultiPoly function at all. maybe add a comment making that more clear?

Reply:
done—thanks!